p11-kit/test-transport.c

```diff
@@ -87,8 +88,9 @@ setup_remote (void *unused)
 	data = "remote: |" BUILDDIR "/p11-kit/p11-kit" EXEEXT " remote " BUILDDIR "/.libs/mock-five" SHLEXT "\nx-init-reserved: initialize-arg";
 	p11_test_file_write (test.user_modules, "init-arg.module", data, strlen (data));
 
-	p11_config_user_modules = test.user_modules;
-	p11_config_user_file = test.user_config;
+	p11_kit_override_system_files( P11_KIT_IGNORE_PTR, test.user_config,
```

Comment:
I meant to add a space before "(", not after it.

Reply:
Done